examples/websockets/streamconn_on_and_off.py

```diff
@@ -48,12 +49,18 @@ def consumer_thread():
                         level=logging.INFO)
 
     loop = asyncio.get_event_loop()
+    pool = ThreadPoolExecutor(1)
 
     while 1:
         try:
-            threading.Thread(target=consumer_thread).start()
+            pool.submit(consumer_thread)
             time.sleep(20)
             loop.run_until_complete(conn.stop_ws())
             time.sleep(20)
-        except:
+        except KeyboardInterrupt:
+            print("Interrupted execution by user")
+            loop.run_until_complete(conn.stop_ws())
+            exit(0)
+        except Exception:
+            # let the execution continue
```

Comment:
At the very least log it! But I'd prefer letting this through.

Reply:
ok